build_misc/templates/autotools-lib.mk

```diff
@@ -0,0 +1,54 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS   += @pkg@
+@PKG@_VERSION := @PKG_VERSION@
+@PKG@_SOVER   := @SOVER@
```

Comment:
Don't use a variable for the SO version, this is only used when the package doesn't provide a SO version itself.